cat_server/helloworld/unikernel.ml

```diff
@@ -3,6 +3,9 @@ open V1_LWT
 
 module Main (C:CONSOLE) = struct
 
-  let start console = OS.Time.sleep 2.0 >>
+  let start console =
+    OS.Time.sleep 2.0
+
```

Comment:
> the lwt scheduler activation- I don't find the >>= fun line noise
> particularly visible.

Thinking in these terms doesn't make much sense to me (that "scheduler activation" thing). You are simply dealing with a monad. A monad is simply a programmable semicolon. Imagine putting a blank line after each semi-colon in an imperative language, it would lead to unreadable functions.